docs/introduction.rst

```diff
@@ -57,7 +57,7 @@ response that you might get from ``receive``::
     {
         "type": "http.response.start",
         "status": 200,
-        "headers": [b"X-Header", b"Amazing Value"],
+        "headers": [[b"X-Header", b"Amazing Value"]],
```

Comment:
List of tuple-pairs no? (I know the spec you linked has `[[]]` but...)

Reply:
It's agnostic to tuples vs lists, I thought I'd keep it simple by using lists of lists

> An iterable of [name, value] two-item iterables

Reply:
Yep, just having seen this conversation go past before, it might be there's an Org preference... https://github.com/django/django/pull/11362

I leave it to you. 😉

Reply:
lists of tuples it is